release_build_files/readme.md

```diff
@@ -571,13 +571,10 @@ code.
 
 ## Release Notes
 
-### 7.3.0
+### 7.2.0
 -   Changes
     -   General (Android): the STLPort support is deprecated and will be removed
         in the next major release. Please [use libc++ instead](https://developer.android.com/ndk/guides/cpp-support#cs).
```

Comment:
"Support for the STLPort STL variant is deprecated..."
Also wrap to 80 (put the link on the next line)?

Reply:
Or just copy the warning text exactly, I like that.

Reply:
> Or just copy the warning text exactly, I like that.

Done.

Put the whole URL on the next line -- I'm not sure if splitting the markdown element in two could break the formatting?

Reply:
I was thinking wrapping it like this:
 -   General (Android): the STLPort support is deprecated and will be removed
        in the next major release. Please
        [use libc++ instead](https://developer.android.com/ndk/guides/cpp-support#cs).